# custom message for xyz_bxbybz values

float64 x
float64 y
float64 z
float64 bx_i
float64 bx_j
float64 bx_k
float64 by_i
float64 by_j
float64 by_k
float64 bz_i
float64 bz_j
float64 bz_k
